### PR TITLE
Feat: next tally settings

### PR DESCRIPTION
**Pull request by ianshade**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature with a change of default behavior.

* **What is the current behavior?** (You can also link to an open issue here)
In DashboardPanel, when establishing if an AdLib is next, in addition to next pieces' tags and ids, tags from unfinished pieces were checked. That (depending on CSS styling) might lead to displaying unfinished pieces as both on-air and next. This can be considered incorrect in most cases, except some specific use cases.

* **What is the new behavior (if this is a feature change)?**
By default, unfinished piece tags will not be taken into account when establishing if a piece is next.
Two options to are added to the Filter DashboardPanel:
  - _Current part can contain next pieces_ - this enables taking unfinished piece tags into account when establishing if a piece is next. It's up to the blueprints and/or the custom CSS to make sure the tally indication is correct.
  - _Indicate only one next piece per source layer_ - if enabled, only the first piece per each sourceLayer will be indicated as next, Piece ids and tags from the next part have priority over unfinished piece tags.

* **Other information**:
Screenshot
image

**Status**

- [ ] Code documentation for the relevant parts in the code have been added/updated by the PR author
- [x] The functionality has been tested by the PR author
- [ ] The functionality has been tested by NRK

### CODE REVIEW

**Review by Julusian — CHANGES_REQUESTED** [review]
Seems reasonable to me
A few bits to tidy, including a git submodule that appears to have been accidentally added

File: meteor/client/ui/Settings/RundownLayoutEditor.tsx [hunk 1]
```diff
@@ -778,6 +780,47 @@ export default translateWithTracker<IProps, IState, ITrackedProps>((props: IProp
...
+										modifiedClassName="bghl"
+										attribute={`filters.${index}.oneNextPerSourceLayer`}
+										obj={item}
+										type="checkbox"
+										collection={RundownLayouts}
+										className="mod mas"
+									/>
+								</label>
+							</div>
+							<div className="mod mvs mhs">
```
[the remark this answers is not in the document]
Comment (ianshade) [reply meteor/client/ui/Settings/RundownLayoutEditor.tsx:810]: Removed, it came from a different feature

File: meteor/client/ui/Shelf/DashboardPanel.tsx [hunk 2]
```diff
@@ -827,20 +836,65 @@ export function isAdLibOnAir(
...
+	nextTags: IDashboardPanelTrackedProps['nextTags'],
+	adLibs: AdLibPieceUi[],
+	nextInCurrentPart: boolean,
+	oneNextPerSourceLayer: boolean
+): Array<AdLibPieceUi & { isNext: boolean }> {
+	const nextAdlibs: Set<PieceId> = new Set()
+	const nextAdlibsPerLayer: Map<string, PieceId> = new Map()
+	adLibs.forEach((adLib) => {
+		if (
+			nextAdLibIds.includes(adLib._id) ||
```
[the remark this answers is not in the document]
Comment (ianshade) [reply meteor/client/ui/Shelf/DashboardPanel.tsx]: Done

File: meteor/client/ui/Shelf/DashboardPanel.tsx [hunk 3]
```diff
@@ -827,20 +836,65 @@ export function isAdLibOnAir(
...
+					nextAdlibsPerLayer.set(adLib.sourceLayerId, adLib._id)
+				}
+			}
+			nextAdlibs.add(adLib._id)
+		}
+	})
+	if (nextInCurrentPart) {
+		adLibs.forEach((adLib) => {
+			if (adLib.nextPieceTags && adLib.nextPieceTags.every((tag) => unfinishedTags.includes(tag))) {
+				if (oneNextPerSourceLayer) {
```
[the remark this answers is not in the document]
Comment (ianshade) [reply meteor/client/ui/Shelf/DashboardPanel.tsx]: Done

**Review by Julusian — APPROVED** [review]

**Pull request merged**